src/t_stream.c

```diff
@@ -785,6 +794,8 @@ int64_t streamTrim(stream *s, streamAddTrimArgs *args) {
             p = lpNext(lp,p); /* Skip all master fields. */
         p = lpNext(lp,p); /* Skip the zero master entry terminator. */
 
+
+	streamID delete_entry_id = {0};
```

Comment:
why declare this here rather than in the scope that uses it?

Reply:
moved to appropriate scope